### ISSUE TITLE
TeachingTip is not read by Windows Accessibility Narrator when IsLightDismissEnabled=True

### ISSUE THREAD

**Issue by g0g8rz** [microsoft/microsoft-ui-xaml#9601]
### Describe the bug

With Narrator you can enable "scan" mode which will allow you to read any text on screen for accessibility reasons.  If you have a TeachingTip and IsLightDismissEnabled=True, Narrator can not read the text.  When the tip is opened, it simply says "Popup Window. Press F6 to go to notification".  But you can't navigate through the tip to read the text.  This works as expected if IsLightDismissEnabled=False.  

### Steps to reproduce the bug

Create an application with a Button that has a teaching tip, for example:
XML:

`<Button x:Name="MyButton" Click="MyButton_OnClick">
    <Button.Resources>
        <TeachingTip x:Name="Tip" 
                               Subtitle="Here is my text to read" Target="{x:Bind MyButton}" 
                               IsLightDismissEnabled="True"/>
    </Button.Resources>
</Button>`

C# Code-behind Click handler

`private void MyButton_OnClick(object sender, RoutedEventArgs e)
    {
        Tip.IsOpen = true;
    }`

1. Run it
2. Turn on Narrator from Settings > Accessibility > Narrator
3. Press Caps Lock > Space to enable scan mode
4. Click the button - You should hear "Popup Window. Press F6 to go to notification"
5. Press F6 - Nothing happens
6. Navigate the TeachingTip in scan mode by pressing up and down arrow keys - Nothing happens

Now do the same thing but with IsLightDismissEnabled=False.  You'll notice that when you press F6, you are now able to press up/down to navigate between the X button and the text.  When the text is highlighted, it will read it out loud.  This is the proper behavior that the TeachingTip should have when IsLightDismissEnabled=True.

### Expected behavior

User should hear the text read when scanning the TeachingTip with up and down arrows

### NuGet package version

WinUI 3 - Windows App SDK 1.5.2: 1.5.240404000

### Windows version

Windows 11 (22H2): Build 22621

**Comment by lauren-ciha** [comment]
Hi @g0g8rz , 

Did you also find that Subtitles weren't narrated?

Here's what happened when I used a Subtitle instead of a Title when IsLightDismissEnabled was False:

https://github.com/user-attachments/assets/429baf9f-0cb8-4f69-b6d8-bf2271a5eebb

I worked around this bug by:
* Setting IsLightDismissEnabled to False and
* Changing the Subtitle to a Title

**Comment by g0g8rz** [comment]
A year later and I ran into this again, completely forgot I filed this bug and stumbled back here by my attempts to fix it again.  I was able to somehow work around the issue by explicitly setting the AutomationProperties.Name to the text I wanted to be read.  However, the Narrator still reads "Press F6 to go to notification" even though this has no effect.  This is really confusing for a visually impaired user and should be fixed.  But given no official response on the ticket in over a year, I'm not confident it ever will be fixed.
